target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -169,9 +169,6 @@ ramips_board_detect() {
 	*"DWR-512 B")
 		name="dwr-512-b"
 		;;
-	*"DWR-922 E2")
```

Comment:
@squigley No, you need to squash the commits, please follow https://openwrt.org/submitting-patches#squashing_commits

Reply:
In this simple case you can probably just do:

```
git reset HEAD^
git commit -vs --amend  target/linux/ramips/base-files/lib/ramips.sh
git push -f
```

Reply:
@squigley Don't worry, just run `git reflog`, find the commit with previous state, then do `git reset --hard <git-hash-good-state>` and try again :-)

Reply:
I hadn't seen this, but I did a "git reset --hard HEAD" and made all my changes again, added, committed, pushed, and made a new PR. #1660